modules/oxide-components/src/main/ts/components/iconbutton/IconButton.tsx

```diff
@@ -1,3 +1,5 @@
+import { forwardRef } from 'react';
```

Comment:
`forwardRef` is [deprecated in React 19](https://react.dev/reference/react/forwardRef). `ref` can be used instead

Reply:
I am well aware, but we are still on react 18. Once we upgrade we will be able to remove all of these forwardRefs